projects/python/cdm/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+
+###############################################################################
+# Policy adjustments
+###############################################################################
+cmake_policy(SET CMP0070 NEW)
+###############################################################################
+# Options
+###############################################################################
+
+###############################################################################
+# Base Variables
+###############################################################################
+set(PROJECT_NAME Python_CommonDataModel)
+set(PREFIX cdm_py)
+
+set(${PREFIX}_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include" )
+set(${PREFIX}_PRIVATE_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src" )
+set(${PREFIX}_GENERATED_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}" )
+set(${PREFIX}_UNIT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/unit" PARENT_SCOPE)
+
+message(STATUS "Configuring ${PROJECT_NAME}")
+###############################################################################
+# Requirments
+###############################################################################
+
+###############################################################################
+#Code Generation
+###############################################################################
+
+add_source_files(SRCS ${CMAKE_CURRENT_SOURCE_DIR}/src "*.py"   "Sources\\")
+add_source_files(SRCS ${CMAKE_CURRENT_BINARY_DIR}/src "*.py"   "Sources\\")
```

Comment:
Do we have ant files in this folder?  If not remove this call.  I don't see anything generating files up to this point